frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -147,17 +147,14 @@
                                                 vm.loaderTitle = '';
                                                 vm.loaderContainer = angular.element('.exist-team-card');
 
-                                                // show loader
-                                                vm.startLoader("Loading Teams");
-                                                // loader end
 
```

Comment:
Now instead of using `startLoader()` function for showing loader I've just set `ng-show` variable to True which was already set to True in line no. 146